### ISSUE TITLE
ROM simulation fails for a zone without windows

### ISSUE THREAD

**Issue by PGorzalka** [RWTH-EBC/AixLib#1545]
If a TEASER model with no window area is exported to AixLib, simulation fails due `eqAirTempWall.sunblind` not being connected from the outside in `ThermalZone`. The reason is that `eqAirTempWall` is conditional to `(sum(zoneParam.AExt) + sum(zoneParam.AWin)) > 0`, but the input from `simpleExternalShading` is conditional to `sum(zoneParam.ATransparent) > 0`.

I suggest removing the conditionality of `simpleExternalShading` and `corGMod` in `AixLib.ThermalZones.ReducedOrder.ThermalZone.ThermalZone`.

**Comment by DaJansenGit** [comment]
Hey @PGorzalka, thanks for posting this issue.

Your approach would work, but wouldn't it be more elegant to make the input `sunblind` conditional based on `sum(zoneParam.ATransparent) > 0` as well in `AixLib.ThermalZones.ReducedOrder.EquivalentAirTemperature.BaseClasses.PartialVDI6007`? In this case the following equation in the model also needs to be conditional:
`` `TEqWin=TDryBul.+delTEqLWWin*(ones(n)-sunblind);``` 


Maybe I'm missing something?

**Comment by PGorzalka** [comment]
Hey @DaJansenGit , this could indeed be a more elegant solution, but `AixLib.ThermalZones.ReducedOrder.EquivalentAirTemperature.BaseClasses.PartialVDI6007` is an IBPSA model, so I'm not sure if there is room for such changes there. Furthermore, the logic of the sunblinds in the model makes it needed for each orientation no matter the weight factors, so that wouldn't be in line with the current way of modelling.

So I'd rather go with the solution proposed in my first post.

**Comment by DaJansenGit** [comment]
Yes, I agree that making those changes regarding the orientation and that it is an IBPSA model is not the best solution. Please go ahead with your proposed solution.
